### PR TITLE
Sentry setup

### PR DESCRIPTION
**Pull request by govuklaurence**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -42,5 +42,13 @@ class Application < Rails::Application
       end
     end
     config.autoload_paths += %W(#{config.root}/app/workers)
+
+    config.filter_parameters << :password
+    config.filter_parameters << :password_confirmation
+
+    Raven.configure do |config|
+        config.dsn = 'https://4bd98ddbad194c9b91e047734b000c69:5c943d0ca981483fa8b6a98cd85f0eaf@sentry.io/236602'
```
Comment (TobyRet) [inline config/application.rb]: Ok in hear or should it be an env_var

Comment (govuklaurence) [reply config/application.rb]: Oh balls good point

**Comment by govuklaurence** [comment]
oh, dammit. bad rebase.

**Comment by govuklaurence** [comment]
Closing b/c bad rebase

**Pull request closed without merging**